src/test/ui/async-await/in-trait/async-associated-types.rs

```diff
@@ -0,0 +1,24 @@
+// edition: 2021
```

Comment:
This should have a bug filed for it I think

Reply:
Filed https://github.com/rust-lang/rust/issues/102681